image/templates/helm/stackrox-central/templates/01-central-13-deployment.yaml

```diff
@@ -108,6 +108,10 @@ spec:
         - name: ROX_ENABLE_OPENSHIFT_AUTH
           value: "true"
         {{- end }}
+        {{- if ._rox.env.openshift }}
+        - name: ROX_OPENSHIFT_API
+          value: "true"
+        {{- end }}
```

Comment:
I'm a bit confused by this change. If you look at the usages of ´ROX_OPENSHIFT_API´cas well as the description of it, you see that its only used within sensor deployments and used for changing the setup of informers there for resources on OpenShift.

I would say we should rather create a separate environment variable that is set, instead of reusing this one as it could be difficult to separate the contexts and meanings when its set (i.e. when set in sensor it means something regarding informers vs if its set in central its used to specify central is running on openshift for telemetry)